Sources/SwiftProtobuf/Google_Protobuf_Any+Extensions.swift

```diff
@@ -49,6 +49,26 @@ extension Google_Protobuf_Any {
     typeURL = buildTypeURL(forMessage:message, typePrefix: typePrefix)
     _storage.state = .message(message)
   }
+  
+  /// Creates a new `Google_Protobuf_Any` by decoding the given string
+  /// containing a serialized message in Protocol Buffer text format.
+  ///
+  /// - Parameters:
+  ///   - textFormatString: The text format string to decode.
+  ///   - extensions: An `ExtensionMap` used to look up and decode any
+  ///     extensions in this message or messages nested within this message's
+  ///     fields.
+  /// - Throws: an instance of `TextFormatDecodingError` on failure.
+  @_disfavoredOverload
+  public init(
```

Comment:
Should we explicitly mark all of these legacy shims as inline?  Would it help keep the library size down if they aren't used?